README.release

```diff
@@ -0,0 +1,45 @@
+= Steps for making a release
+
+1. Get the maintainer's tools:
+
+```
+yum install autoconf automake autoconf-archive make
+```
+
+2. Get `gitlog-to-changelog` tool:
+
+```
+wget https://raw.githubusercontent.com/manuelbua/gitver/master/gitlog-to-changelog
+chmod a+x gitlog-to-changelog
+PATH="$(pwd):$PATH"
+```
+
+3. Git release administrative -- a commit like this: https://github.com/devexp-db/postgresql-setup/commit/3cf4aaa5
+
+4. Add a tag for the release
+
+```
+git tag -a v8.6 -m "Release v8.6"
+```
+
+5. Post-release administrative -- a commit like this: https://github.com/devexp-db/postgresql-setup/commit/c7ed7144
+
+6. Push the changes
+
+```
+git push
+```
+
+7. Create a tarball
+```
+aclocal
+automake --add-missing
+autoconf
```

Comment:
autoreconf (-vfi) should do this

Reply:
autoreconf (-vfi) should do this

Reply:
Thanks for the suggestion, will use this.